bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/di/DefaultModelServerModule.java

```diff
@@ -94,6 +99,10 @@ protected void configure() {
       bind(UriHelper.class).to(bindUriHelper()).in(Singleton.class);
       bind(TransactionController.class).to(bindTransactionController()).in(Singleton.class);
 
+      // Not a singleton
+      Optional.ofNullable(bindJsonPatchHelper()).filter(not(JsonPatchHelper.class::equals))
```

Comment:
This isn't really obvious. Since Guice supports implicit bindings, I think I'd prefer that (i.e. remove the binding entirely, since the class can already be instantiated and has an @Inject annotation; Guice can figure it out).

Custom Modules can declare their own binding if they need to customize/subclass the JsonPatchHelper

Reply:
Makes sense.